setup.py

```diff
@@ -61,4 +61,5 @@ def get_install_requirements():
     include_package_data=True,
     packages=setuptools.find_packages(where="./src"),
     python_requires=">= 3.9",
+    package_data={"mpyl": ["py.typed"]},
```

Comment:
maybe you can assign `setuptools.find_packages(where="./src")` to a variable and use that both here and in `packages`?

Reply:
no, we just need to include the py.typed file here

Reply:
I'm referring to the key `"mpyl"`, in the docs of mypy those seem to be the same but I don't actually know what `setuptools.find_packages(where="./src")` returns. 

```
    package_data={"package_b": ["py.typed", "lib.pyi"]},
    packages=["package_b"]
```

Reply:
I think these are two different things. `packages` is the list of packages in the whole mpyl library. `package_data` is a mapping from a package to extra files it should include.